Tools/MonoGame.Effect.Compiler/Effect/Spirv/Types/SpirvTypeArray.cs

```diff
@@ -0,0 +1,33 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System.Diagnostics;
+
+namespace MonoGame.Effect.Compiler.Effect.Spirv
+{
+    public class SpirvTypeArray : SpirvTypeBase
```

Comment:
Seems like all these classes are internal to the Effect compiler system and shouldn't be `public`.